packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -66,88 +96,51 @@ const validateOperationTask = (task, props) => {
   })
 }
 
-// Note: `bypassVdiChainsCheck` must be enabled because the XAPI might be not
-// able to coalesce VDIs as fast as the tests run.
-//
-// See https://xen-orchestra.com/docs/backup_troubleshooting.html#vdi-chain-protection
 describe('backupNg', () => {
-  let defaultBackupNg
-
-  beforeAll(() => {
-    defaultBackupNg = {
-      name: 'default-backupNg',
-      mode: 'full',
-      vms: {
-        id: config.vms.default,
-      },
-      settings: {
-        '': {
-          reportWhen: 'never',
-        },
-      },
-    }
-  })
-
   describe('.createJob() :', () => {
     it('creates a new backup job without schedules', async () => {
-      const backupNg = await xo.createTempBackupNgJob(defaultBackupNg)
-      expect(backupNg).toMatchSnapshot({
-        id: expect.any(String),
-        userId: expect.any(String),
-        vms: expect.any(Object),
-      })
-      expect(backupNg.vms).toEqual(defaultBackupNg.vms)
-      expect(backupNg.userId).toBe(xo._user.id)
+      const input = {
+        mode: 'full',
+        vms: {
+          id: config.vms.default,
+        },
+      }
+      const output = await xo.createTempBackupNgJob(input)
+      validateBackupJob(xo, input, output)
     })
 
     it('creates a new backup job with schedules', async () => {
       const scheduleTempId = randomId()
-      const { id: jobId } = await xo.createTempBackupNgJob({
-        ...defaultBackupNg,
+      const input = {
+        mode: 'full',
         schedules: {
           [scheduleTempId]: DEFAULT_SCHEDULE,
         },
         settings: {
-          ...defaultBackupNg.settings,
           [scheduleTempId]: { snapshotRetention: 1 },
         },
-      })
-
-      const backupNgJob = await xo.call('backupNg.getJob', { id: jobId })
-
-      expect(backupNgJob).toMatchSnapshot({
-        id: expect.any(String),
-        userId: expect.any(String),
-        settings: expect.any(Object),
-        vms: expect.any(Object),
-      })
-      expect(backupNgJob.vms).toEqual(defaultBackupNg.vms)
-      expect(backupNgJob.userId).toBe(xo._user.id)
-
-      expect(Object.keys(backupNgJob.settings).length).toBe(2)
-      const schedule = await xo.getSchedule({ jobId })
-      expect(typeof schedule).toBe('object')
-      expect(backupNgJob.settings[schedule.id]).toEqual({
-        snapshotRetention: 1,
-      })
-
-      expect(schedule).toMatchSnapshot({
-        id: expect.any(String),
-        jobId: expect.any(String),
-      })
+        vms: {
+          id: config.vms.default,
+        },
+      }
+      const output = await xo.createTempBackupNgJob(input)
+      validateBackupJob(xo, input, output)
     })
   })
 
   describe('.delete() :', () => {
     it('deletes a backup job', async () => {
       const scheduleTempId = randomId()
       const { id: jobId } = await xo.call('backupNg.createJob', {
-        ...defaultBackupNg,
+        mode: 'full',
+        name: `xo-server-test ${new Date().toISOString()}`,
```

Comment:
Why not use the default value?

Reply:
Should i move `getDefaultName` to a dedicated module named `_defaultValues`?

Reply:
As you whish, but my question was, why is it explicit and not using the default value here?

Reply:
It's my bad.

Reply:
Oh, I understand now.